capsicum/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "capsicum"
+edition = "2021"
+version = "0.1.3"
+authors = ["Daniel Robertson <dan.robertson@anidata.org>"]
```

Comment:
Not related, but I need to update this... That email definitely doesn't work now :p